### PR TITLE
New raw_data_loader func and extractor integration for *Camera.frameData.bin files

### PR DESCRIPTION
**Pull request by nbonacchi**

### CODE REVIEW

**Comment by k1o0** [comment]
We should review the dataset type first.  For one the name should be in Haskell case, i.e. 'leftCamera.frameData.bin'.  The extractors and CI tests will need updating before any sessions are acquired with this dataset.

**Comment by nbonacchi** [comment]
Sure that's just there for when we get the release candate data to be able to look at it...
Now that I look at it this should be a draft PR not a full PR...

File: ibllib/io/raw_data_loaders.py [hunk 1]
```diff
@@ -103,16 +103,75 @@ def load_data(session_path: Union[str, Path], time='absolute'):
...
+            }
+    """
+    camera = assert_valid_label(camera)
+    fpath = Path(session_path).joinpath("raw_video_data", f"_iblrig_{camera}Camera.frameData.bin")
+    assert fpath.exists(), f"{fpath}\nFile not Found: Could not find bin file for cam <{camera}>"
+    rdata = np.fromfile(fpath, dtype=np.float64)
+    assert len(rdata) % 4 == 0, "Dimension mismatch: bin file length is not mod 4"
+    rows = int(len(rdata) / 4)
+    data = np.reshape(rdata.astype(np.int64), (rows, 4))
+    df_dict = {
```
Comment (k1o0) [inline ibllib/io/raw_data_loaders.py:140]: Could use fromkeys for fewer lines :)

File: ibllib/io/raw_data_loaders.py [hunk 2]
```diff
@@ -103,16 +103,75 @@ def load_data(session_path: Union[str, Path], time='absolute'):
...
+        "Timestamp": None,
+        "embeddedTimeStamp": None,
+        "embeddedFrameCounter": None,
+        "embeddedGPIOPinState": None
+    }
+    df = pd.DataFrame(data, columns=df_dict.keys())
```
Comment (k1o0) [inline ibllib/io/raw_data_loaders.py:146]: Could add a return here as there's no need to parse the df if raw is True

**Review by k1o0 — APPROVED** [review]

**Comment by nbonacchi** [comment]
OK integrated your suggestions, ready for merge!

**Pull request merged**